www/docs/src/components/StudioHeading.astro

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
Now _this_ is a fuckin header component bloody hell

Reply:
Stolen straight from Astro Docs!

Reply:
I was going to say, this looks familiar! 😄